### DISCUSSION TITLE
Structure of test_x and test_y files.

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#186] [BaiduKDDCup2022] [answered]
What is the correct structures of test_x files and test_y files for evaluation.py with the new changes.

0001in.csv and 0002in.csv have to be in two zip files or how?

Thank you

**Comment by ramber1836** [comment, marked as answer]
> What is the correct structures of test_x files and test_y files for evaluation.py with the new changes.
> 
> 0001in.csv and 0002in.csv have to be in two zip files or how?
> 

Yes, they have to be in two zip files.
